packages/cockpit/src/transactions/conversion/boleto/result.js

```diff
@@ -0,0 +1,43 @@
+import {
+  always,
+  apply,
+  applySpec,
+  divide,
+  equals,
+  ifElse,
+  juxt,
+  last,
+  multiply,
+  path,
+  pipe,
+} from 'ramda'
+
+const total = path(['aggregations', 'boleto', 'doc_count'])
+
+const paid = path(['aggregations', 'boleto', 'paid', 'doc_count'])
+
+const conversion = pipe(
+  juxt([
+    paid,
+    total,
+  ]),
+  ifElse(
+    pipe(
```

Comment:
Essa função pode ser separada em outra de nome `lastEquals`:

```js
const lastEquals = value => pipe(
  last,
  equals(value)
)
```

Ficando a chamada:

```suggestion
    lastEquals(0)
```